src/platforms/esp32/test/main/test_main.c

```diff
@@ -489,13 +494,17 @@ TEST_CASE("test_ssl", "[test_run]")
     TEST_ASSERT(ret_value == OK_ATOM);
 }
 
+// Works C3 on local runs, but fails GH actions
+#if !CONFIG_IDF_TARGET_ESP32C3
 TEST_CASE("test_rtc_slow", "[test_run]")
 {
     term ret_value = avm_test_case("test_rtc_slow.beam");
     TEST_ASSERT(term_to_int(ret_value) == 0);
 }
+#endif
 
-#if ESP_IDF_VERSION_MAJOR >= 5
+// Works C3 on local runs, but fails GH actions
```

Comment:
just to understand, how it fails?

Reply:
2024-03-13 08:48:53 
2024-03-13 08:48:55 E (15949) task_wdt: Task watchdog got triggered. The following tasks/users did not reset the watchdog in time:
2024-03-13 08:48:55 E (15949) task_wdt:  - IDLE (CPU 0)
2024-03-13 08:48:55 E (15949) task_wdt: Tasks currently running:
2024-03-13 08:48:55 
2024-03-13 08:48:55 E (15949) task_wdt: CPU 0: IDLE
2024-03-13 08:48:55 E (15949) task_wdt: Print CPU 0 (current core) backtrace
2024-03-13 08:48:55 Print CPU 0 (current core) registers
2024-03-13 08:48:55 Core  0 register dump:
2024-03-13 08:48:55 
2024-03-13 08:48:55 MEPC    : 0x40386c7e  RA      : 0x4208c0b0  SP      : 0x3fca1b80  GP      : 0x3fc97200  
2024-03-13 08:48:55 
2024-03-13 08:48:55 TP      : 0x3fc47d94  T0      : 0x00000000  T1      : 0x00000000  T2      : 0x00000000  
2024-03-13 08:48:55 S0/FP   : 0x00000008  S1      : 0x00000001  A0      : 0x00000001  A1      : 0x3fca1b5f  
2024-03-13 08:48:55 
2024-03-13 08:48:55 A2      : 0x00000001  A3      : 0x3fca1be4  A4      : 0x0000001c  A5      : 0x00000000  
2024-03-13 08:48:55 A6      : 0x00000000  A7      : 0x00000000  S2      : 0x00000000  S3      : 0x00000000  
2024-03-13 08:48:55 S4      : 0x00000000  S5      : 0x00000000  S6      : 0x00000000  S7      : 0x00000000  
2024-03-13 08:48:55 S8      : 0x00000000  S9      : 0x00000000  S10     : 0x00000000  S11     : 0x00000000  
2024-03-13 08:48:55 T3      : 0x00000000  T4      : 0x00000000  T5      : 0x00000000  T6      : 0x00000000  
2024-03-13 08:48:55 
2024-03-13 08:48:55 MSTATUS : 0x00000000  MTVEC   : 0x00000000  MCAUSE  : 0x00000000  MTVAL   : 0x4038b836  
2024-03-13 08:48:55 MHARTID : 0x00000000  
2024-03-13 08:48:55 
2024-03-13 08:48:58 Guru Meditation Error: Core  0 panic'ed (Interrupt wdt timeout on CPU0). 
2024-03-13 08:48:58 
2024-03-13 08:48:58 
2024-03-13 08:48:58 Core  0 register dump:
2024-03-13 08:48:58 MEPC    : 0x40386c7e  RA      : 0x4208c0b0  SP      : 0x3fca1b80  GP      : 0x3fc97200  
2024-03-13 08:48:58 TP      : 0x3fc47d94  T0      : 0x00000000  T1      : 0x00000000  T2      : 0x00000000  
2024-03-13 08:48:58 S0/FP   : 0x00000008  S1      : 0x00000001  A0      : 0x00000001  A1      : 0x3fca1b5f  
2024-03-13 08:48:58 A2      : 0x00000001  A3      : 0x3fca1be4  A4      : 0x0000001c  A5      : 0x00000000  
2024-03-13 08:48:58 A6      : 0x00000000  A7      : 0x00000000  S2      : 0x00000000  S3      : 0x00000000  
2024-03-13 08:48:58 
2024-03-13 08:48:58 S4      : 0x00000000  S5      : 0x00000000  S6      : 0x00000000  S7      : 0x00000000  
2024-03-13 08:48:58 S8      : 0x00000000  S9      : 0x00000000  S10     : 0x00000000  S11     : 0x00000000  
2024-03-13 08:48:58 T3      : 0x00000000  T4      : 0x00000000  T5      : 0x00000000  T6      : 0x00000000  
2024-03-13 08:48:58 MSTATUS : 0x00001880  MTVEC   : 0x40380001  MCAUSE  : 0x00000018  MTVAL   : 0x00000000  
2024-03-13 08:48:58 MHARTID : 0x00000000  
2024-03-13 08:48:58 
2024-03-13 08:48:58 
2024-03-13 08:48:58 Stack memory:
2024-03-13 08:48:58 3fca1b80: 0x00000000 0x00000000 0x00000000 0x4038b836 0x00000000 0x00000000 0x00000000 0x4038aaee
2024-03-13 08:48:58 3fca1ba0: 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000
2024-03-13 08:48:58 3fca1bc0: 0x00000000 0xa5a5a5a5 0xa5a5a5a5 0xa5a5a5a5 0xa5a5a5a5 0xa5a5a5a5 0xa5a5a5a5 0xa5a5a5a5
2024-03-13 08:48:58 3fca1be0: 0x00000150 0x3fca1b00 0x00000000 0x3fc99c64 0x3fc99c64 0x3fca1be4 0x3fc99c5c 0x00000019
2024-03-13 08:48:58 3fca1c00: 0x00000000 0x00000000 0x3fca1be4 0x00000000 0x00000000 0x3fca15e0 0x454c4449 0x00000000
2024-03-13 08:48:58 3fca1c20: 0x00000000 0x00000000 0x3fca1bd0 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000
2024-03-13 08:48:58 3fca1c40: 0x3fc9ecec 0x3fc9ed54 0x3fc9edbc 0x00000000 0x00000000 0x00000001 0x00000000 0x00000000
2024-03-13 08:48:58 3fca1c60: 0x00000000 0x420054c4 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000
2024-03-13 08:48:58 3fca1c80: 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000
2024-03-13 08:48:58 
2024-03-13 08:48:58 3fca1ca0: 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000
2024-03-13 08:48:58 3fca1cc0: 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000
2024-03-13 08:48:58 3fca1ce0: 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000
2024-03-13 08:48:58 3fca1d00: 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000
2024-03-13 08:48:58 3fca1d20: 0x00000000 0x00000000 0x00000000 0x00000000 0x00000200 0x0000000c 0x094e000a 0x00000000
2024-03-13 08:48:58 3fca1d40: 0x3fca1e0c 0x00000054 0x00000000 0x3fca1d48 0x00000000 0x00000000 0x00000000 0x3fca1d60
2024-03-13 08:48:58 
2024-03-13 08:48:58 3fca1d60: 0xffffffff 0x3fca1d60 0x3fca1d60 0x00000000 0x3fca1d74 0xffffffff 0x3fca1d74 0x3fca1d74
2024-03-13 08:48:58 3fca1d80: 0x00000001 0x00000001 0x00000000 0x0000ffff 0x00000000 0x00000000 0x3fca1d40 0x00000054
2024-03-13 08:48:58 3fca1da0: 0x00000000 0x3fca1da0 0x00000000 0x00000000 0x00000000 0x3fca1db8 0xffffffff 0x3fca1db8
2024-03-13 08:48:58 3fca1dc0: 0x3fca1db8 0x00000000 0x3fca1dcc 0xffffffff 0x3fca1dcc 0x3fca1dcc 0x00000001 0x00000001
2024-03-13 08:48:58 3fca1de0: 0x00000000 0x0000ffff 0x00000000 0x00000000 0x3fca1d98 0x00000010 0x3fc9a888 0x3fca1e1c
2024-03-13 08:48:58 3fca1e00: 0x00000001 0x00000000 0x0000000c 0x08520008 0x00000000 0x3fcac39c 0x00000010 0x00000000
2024-03-13 08:48:58 3fca1e20: 0x3fca1be4 0x00000000 0x00000000 0x00000154 0x3fca1e84 0x3fca1e8c 0x3fca1f84 0x3fca1e84
2024-03-13 08:48:58 
2024-03-13 08:48:58 3fca1e40: 0x00000000 0x3fca1e48 0xffffffff 0x3fca1e48 0x3fca1e48 0x00000000 0x3fca1e5c 0xffffffff
2024-03-13 08:48:58 
2024-03-13 08:48:58 3fca1e60: 0x3fca1e5c 0x3fca1e5c 0x00000001 0x00000040 0x00000004 0x0000ffff 0x00000000 0x00000000
2024-03-13 08:48:58 3fca1e80: 0x00000000 0x3fca1f88 0x3fca1f88 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000
2024-03-13 08:48:58 3fca1ea0: 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000
2024-03-13 08:48:58 3fca1ec0: 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000
2024-03-13 08:48:58 3fca1ee0: 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000
2024-03-13 08:48:58 3fca1f00: 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000
2024-03-13 08:48:58 3fca1f20: 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000
2024-03-13 08:48:58 3fca1f40: 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000
2024-03-13 08:48:58 3fca1f60: 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000
2024-03-13 08:48:58 
2024-03-13 08:48:58 
2024-03-13 08:48:58 
2024-03-13 08:48:58 
2024-03-13 08:48:58 ELF file SHA256: 6c7287a9f
2024-03-13 08:48:58 
2024-03-13 08:48:58 Rebooting...

Reply:
task_wdt - succeeds local qemu macos - succeeds "act"/docker arm64 - fails gh action.